.editorconfig

```diff
@@ -0,0 +1,21 @@
+# EditorConfig helps developers define and maintain consistent
```

Comment:
It's required to have an ASF header for this file.

Reply:
Sure, Added.

Reply:
> It's required to have an ASF header for this file.

Sure, Added.